app/assets/javascripts/services/miq_service.js

```diff
@@ -158,4 +158,20 @@ ManageIQ.angular.app.service('miqService', ['$timeout', '$document', '$q', 'API'
       miqService.sparkleOff();
     }
   };
+
+  this.redirectBack = function(message, flashType, redirectUrl, reload) {
+    var flash = { message: message };
+
+    flash.level = flashType;
+
+    miqFlashLater(flash);
+
+    $timeout(function() {
+      $window.location.href = redirectUrl;
+
+      if(reload) {
```

Comment:
Assigning to `location.href` already realoads the whole page. We don't have any partial refresh in that area, and it would be impossible to achieve this way anyway.

If this really has an effect (I can't see it), it probably is a timing issue, and some part of the code using this is not waiting for the API request to actually finish.

If I rewrite the function to 

```
    miqFlashLater({
      message: message,
      level: flashType,
    });

    $window.location.href = redirectUrl;
```

it still works exactly the same for me, and I can see the tree updating just fine :).

Please re-check if you're still seeing the issue :).